ads/_ping_.js

```diff
@@ -21,7 +21,21 @@ import {dev} from '../src/log';
  * @param {!Object} data
  */
 export function _ping_(global, data) {
+  //validateData(data, [], ['adWidth', 'adHeight', 'valid']);
   global.document.getElementById('c').textContent = data.ping;
+  global.ping = Object.create(null);
```

Comment:
We already take care of this. https://github.com/ampproject/amphtml/blob/master/3p/integration.js#L170
ad type `_ping_` is only registered in test and localDev mode.